src/AutomatedCar/Models/PlayerCar.cs

```diff
@@ -0,0 +1,23 @@
+﻿using AutomatedCar.SystemComponents;
+using Avalonia;
+using Avalonia.Media;
+using System;
+using System.Collections.Generic;
+
+using System.Text;
+
+namespace AutomatedCar.Models
+{
+    public class PlayerCar : AutomatedCar
```

Comment:
Nem teljesen értem, hogy ez miért kell.

Reply:
Meg akartam két autót jeleníteni, és a sima Car objektumnak nem találtam geomerty tulajdonságát, amin el is hasalt a megjelenítés. Ebből gondoltam, hogy az automatedCar a típusa az npc autónak is, és valahogy meg akartam különböztetni a controlledCar-t a többitől. Ezt a xaml-ben egy saját típussal tudtam megoldani.

Reply:
Végül abban maradtunk hogy a playerCar-t törlöm, és a propertijeit átteszem az AutomatedCar-ba.